### ISSUE TITLE
[FEATURE REQUEST]: Installer

### ISSUE THREAD

**Issue by validity-check** [validity-check/prime#3]
What I think would be useful is an installer which adds prime to the path environment variable so it can be used easier.

It would have to consider updates, uninstalling etc.

**Comment by avinashupadhya99** [comment]
Have you considered `npm`? I can help with publishing it as an `npm` package.

**Comment by validity-check** [comment]
I hadn't thought about that. I think it would be great to publish an npm package.
Shall I assign you?

**Comment by Rebaiahmed** [comment]
Yes , we can publish it to npm and improve the readme file :) i'm interested did you want to assign me ?

**Comment by avinashupadhya99** [comment]
Yes, you can assign this to me.
